build_files/base/copr-repos.sh

```diff
@@ -20,15 +20,22 @@ if [ ${FEDORA_MAJOR_VERSION} -eq "39" ]; then
     rpm-ostree install ptyxis
 fi
 
-# 40 gets only Ptyxis
+# 40 gets Ptyxis and patched Mutter
 if [ ${FEDORA_MAJOR_VERSION} -eq "40" ]; then
     rpm-ostree override replace \
     --experimental \
     --from repo=copr:copr.fedorainfracloud.org:ublue-os:staging \
         vte291 \
         vte-profile
     rpm-ostree install ptyxis
+    if [[ "${IMAGE_FLAVOR}" =~ "bluefin" ]]; then
```

Comment:
I'm not sure IMAGE_FLAVOR is what we're needing.  Checking

Reply:
Wouldn't we want to use brand_name?

Reply:
Let's use BASE_IMAGE_NAME?  This is either silverblue or kinoite
https://github.com/ublue-os/bluefin/blob/main/.github/workflows/reusable-build.yml#L64-L68